.buildbot.sh

```diff
@@ -6,12 +6,14 @@ export CARGO_HOME="`pwd`/.cargo"
 export RUSTUP_HOME="`pwd`/.rustup"
 
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
-sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain nightly -y --no-modify-path
-
+sh rustup.sh -y --no-modify-path
 export PATH=`pwd`/.cargo/bin/:$PATH
 
+rustup toolchain install nightly
 rustup component add --toolchain nightly rustfmt-preview || cargo +nightly install --force rustfmt-nightly
-
 cargo +nightly fmt --all -- --check
```

Comment:
If we only test with stable shouldn't we also use stable to run `cargo fmt`?

Reply:
Unfortunately our settings currently don't work with stable `cargo fmt` :(